hack/pin-bundle-images.sh

```diff
@@ -17,7 +17,7 @@ if [ -n "$DOCKERFILE" ]; then
 fi
 
 #loop over each openstack-k8s-operators go.mod entry
-for MOD_PATH in $(go list -mod=readonly -m -json all | jq -r '. | select(.Path | contains("openstack")) | .Replace // . |.Path' | grep -v openstack-operator | grep -v lib-common); do
+for MOD_PATH in $(go list -mod=readonly -m -json all | jq -r '. | select(.Path | contains("openstack")) | .Replace // . |.Path' | grep -v openstack-operator | grep -v lib-common | grep -v telemetry-operator$); do
```

Comment:
Why is this change needed?

Reply:
I added telemetry-operator to go.mod (not telemetry-operator/api). This totally confused the rest of the code. This change is there to remove this new dependency. You can see the error here: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openstack-k8s-operators_openstack-operator/643/pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl/1754392910935101440

Reply:
This change isn't needed in the latest version anymore and it was removed.